manager/notifier.go

```diff
@@ -231,6 +231,100 @@ func (n *notifier) sendHipChatNotification(op notificationOp, config *pb.HipChat
 	return nil
 }
 
+// Slack body request
+type slackReq struct {
+	Channel     string            `json:"channel,omitempty"`
+	Attachments []slackAttachment `json:"attachments"`
+}
+
+// Slack message attachments to display richly-formatted message blocks
+type slackAttachment struct {
+	Fallback  string                 `json:"fallback"`
+	Pretext   string                 `json:"pretext,omitempty"`
+	Title     string                 `json:"title,omitempty"`
+	TitleLink string                 `json:"title_link,omitempty"`
+	Text      string                 `json:"text"`
+	Color     string                 `json:"color,omitempty"`
+	MrkdwnIn  []string               `json:"mrkdwn_in,omitempty"`
+	Fields    []slackAttachmentField `json:"fields,omitempty"`
+}
+
+// Slack fields to display in a table inside the message attachment
```

Comment:
Strictly speaking, godoc style would be `slackAttachment is ...`. As those are not exported I don't feel strong about this.
They should end with a `.`, though.